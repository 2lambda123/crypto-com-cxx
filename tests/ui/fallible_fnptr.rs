#[cxx::bridge]
mod ffi {
    extern "C++" {
        fn f(callback: fn() -> Result<()>);
    }
}

fn main() {}
