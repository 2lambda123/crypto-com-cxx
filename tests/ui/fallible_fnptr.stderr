error: function pointer returning Result is not supported yet
 --> $DIR/tests/ui/fallible_fnptr.rs:4:24
  |
4 |         fn f(callback: fn() -> Result<()>);
  |                        ^^^^^^^^^^^^^^^^^^
